lib/views/scannerview.dart

```diff
@@ -1,34 +1,56 @@
 import 'package:flutter/material.dart';
 // ignore: depend_on_referenced_packages
 import 'package:flutter_barcode_scanner/flutter_barcode_scanner.dart';
+import 'package:library_qr/Repositry/student_repositry.dart';
 
-class Scannscreen extends StatefulWidget {
-  const Scannscreen({super.key});
+class ScanScreen extends StatefulWidget {
+  const ScanScreen({super.key});
 
   @override
-  // ignore: library_private_types_in_public_api
-  _ScannerScreenState createState() => _ScannerScreenState();
+  ScanScreenState createState() => ScanScreenState();
 }
```

Comment:
warning of using public api in private class.
That is  what made the below changes